tasklib/tests.py

```diff
@@ -875,8 +880,9 @@ def test_setting_up_from_modified_hook_input(self):
 
     def test_export_data(self):
         t = Task(self.tw, description="test task",
-            project="Home",
-            due=pytz.utc.localize(datetime.datetime(2015,1,1,23,23,23)))
+                 project="Home",
+                 due=pytz.utc.localize(datetime.datetime(2015, 1, 1,
+                                                         23, 23, 23)))
```

Comment:
Instead of breaking up the parentheses that are "smaller" here, it's best to go back to the "larger" ones and break them up first (as it often means you can keep related things on the same line). In this case, we can also put each argument to `Task` on its own line, and use single quotes too:

```python
t = Task(
    self.tw,
    description='test task',
    project='Home',
    due=pytz.utc.localize(
        datetime.datetime(2015, 1, 1, 23, 23, 23),
    ),
)
```

Note: I don't know for sure if the `localize()` line would all fit inside 80 characters, so I broke it up just to show how it can be done.

Reply:
Fixed in 8f4fabf